$MirOS: ports/print/ghostscript/afpl/patches/patch-lib_gs_init_ps,v 1.2 2005/06/27 20:14:22 tg Exp $
$OpenBSD: patch-lib_gs_init_ps,v 1.3 2004/05/23 18:51:59 sturm Exp $
--- lib/gs_init.ps.orig	Tue Feb 21 21:28:44 2006
+++ lib/gs_init.ps	Thu Jun 15 18:12:52 2006
@@ -1867,6 +1867,10 @@ initgraphics
 gsave
 
 % Define some control sequences as no-ops.
+
+% unofficial patch:
+% make it possible to print from within acroread pdf viewer
+/Default currenthalftone /Halftone defineresource pop
 % This is a hack to get around problems
 % in some common PostScript-generating applications.
 <04> cvn JOBSERVER {
